apps/web/src/pages/JobPosterTemplates/JobPosterTemplatesPage/JobPosterTemplatesPage.tsx

```diff
@@ -0,0 +1,559 @@
+import RectangleStackIcon from "@heroicons/react/24/outline/RectangleStackIcon";
+import { defineMessage, IntlShape, useIntl } from "react-intl";
+import { FormProvider, useForm } from "react-hook-form";
+import { useQuery } from "urql";
+import { useCallback, useEffect, useMemo } from "react";
+import { useSearchParams } from "react-router-dom";
+import debounce from "lodash/debounce";
+
+import {
+  Button,
+  CardBasic,
+  Heading,
+  Link,
+  Loading,
+  PreviewList,
+  Separator,
+  Sidebar,
+  Well,
+  type PreviewMetaData,
+} from "@gc-digital-talent/ui";
+import {
+  Classification,
+  graphql,
+  LocalizedPoolStream,
+  Maybe,
+  PoolStream,
+  SupervisoryStatus,
+} from "@gc-digital-talent/graphql";
+import {
+  Checklist,
+  Input,
+  localizedEnumToOptions,
+} from "@gc-digital-talent/forms";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import {
+  commonMessages,
+  getLocale,
+  getLocalizedName,
+} from "@gc-digital-talent/i18n";
+
+import Hero from "~/components/Hero";
+import SEO from "~/components/SEO/SEO";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import useRoutes from "~/hooks/useRoutes";
+import processMessages from "~/messages/processMessages";
+
+interface FormValues {
+  keyword: string;
+  classifications: string[];
+  supervisoryStatuses: SupervisoryStatus[];
+  streams: PoolStream[];
+}
+
+const defaultValues = {
+  keyword: "",
+  classifications: [],
+  supervisoryStatuses: [],
+  streams: [],
+} satisfies FormValues;
+
+const pageTitle = defineMessage({
```

Comment:
The page has a different title in the breadcrumbs.  Also, could you please move this out to a .ts file where I can import it for the breadcrumb of the child page? I have it in a message.ts file in the stub in my PR.
![image](https://github.com/user-attachments/assets/1999d56f-1659-497e-bb76-e11c7527aba9)

Reply:
Hmm, not a fan of making breadcrumb text different for a couple reasons but I guess it is close enough :confused:  Updated in [extract messages out, update breadcrumb text](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11530/commits/e7aa2a47b0909eccbd57637c2511c2c4d7870988)

Reply:
Yeah... for what it's worth I agree.

Reply:
I have this translated, by the way, in my PR.
```
  "k5Jueg": {
    "defaultMessage": "Modèles d'emplois",
    "description": "Title for the 'job poster templates' page"
  },
```